backend/src/main/java/ai/verta/modeldb/dataset/DatasetDAORdbImpl.java

```diff
@@ -345,9 +347,10 @@ public Boolean deleteDatasets(List<String> datasetIds) throws InvalidProtocolBuf
       for (DatasetEntity datasetObj : datasetEntities) {
         session.delete(datasetObj);
       }
+      transaction.commit();
```

Comment:
this is better suited in other batch of PRs related to delete?

Reply:
reverted